### PR TITLE
Feature/source detail

### PR DESCRIPTION
**Pull request by XanderVertegaal**
Adds a source detail view and makes sure episodes are queryable on SourceType.

The new/edit form and the associated mutations will be added in other PRs.

### CODE REVIEW

File: backend/person/types/AgentDescriptionType.py [hunk 1]
```diff
@@ -40,4 +40,4 @@ def resolve_describes(
     def resolve_person_references(
         parent: AgentDescription, info: ResolveInfo
     ) -> QuerySet[PersonReference]:
-        return parent.person_references.all()  # type: ignore
+        return PersonReference.objects.filter(person=parent)
```
Comment (XanderVertegaal) [inline backend/person/types/AgentDescriptionType.py:43]: This removes the `# type: ignore`. (No functional change.)

File: frontend/src/app/data-entry/source/source.component.scss [hunk 2]
```diff
@@ -0,0 +1,4 @@
+.entity-item:not(:last-child)::after {
+    // Character \00A0 is a non-breaking space.
+    content: ",\00A0";
+}
```
Comment (lukavdplas) [inline frontend/src/app/data-entry/source/source.component.scss:4]: Oh neat, this is much cleaner than my solution! :)

Perhaps we could generalise this into a more general rule that we can reuse in the application? E.g. `.inline-list > .inline-list-item`

File: frontend/src/app/data-entry/source/source.component.spec.ts [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+import { SourceComponent } from "./source.component";
+
+describe("SourceComponent", () => {
+    let component: SourceComponent;
+    let fixture: ComponentFixture<SourceComponent>;
+
+    beforeEach(() => {
+        TestBed.configureTestingModule({
+            declarations: [SourceComponent],
```
Comment (lukavdplas) [inline frontend/src/app/data-entry/source/source.component.spec.ts:11]: I think this needs to import the `SharedTestingModule`

**Review by lukavdplas — APPROVED** [review]
Neat!

I added some minor suggestions, but none of them are hard requirements.

File: frontend/src/app/data-entry/source/source.component.scss (hunk 2 above)
Comment (XanderVertegaal) [reply frontend/src/app/data-entry/source/source.component.scss:4]: Good idea!

File: frontend/src/app/data-entry/source/source.component.html [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+                        <span>chapter {{ episode.chapter }}</span>
+                    </small>
+                </div>
+                <lc-action-button-group
+                    [editLink]="['episode', episode.id]"
+                    (delete)="deleteEpisode(episode.id)"
+                    [showButtonText]="false"
+                />
+            </div>
+        </div>
```
[the remark this answers is not in the document]
Comment (XanderVertegaal) [reply frontend/src/app/data-entry/source/source.component.html:38]: Good tip! Done!

**Pull request merged**